.github/workflows/deployment.yaml

```diff
@@ -0,0 +1,52 @@
+name: deployment
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+jobs:
+  checks:
+    if: github.event_name != 'push'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Test Build
+        run: |
+          if [ -e yarn.lock ]; then
+            yarn install --frozen-lockfile
+          elif [ -e package-lock.json ]; then
+            npm ci
+          else
+            npm i
+          fi
+          yarn build
+  gh-release:
+    if: github.event_name != 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - uses: webfactory/ssh-agent@v0.5.0
+        with:
+          ssh-private-key: ${{ secrets.GH_PAGES_DEPLOY }}
+      - name: Release to GitHub Pages
+        env:
+          USE_SSH: true
+          GIT_USER: git
+        run: |
+          git config --global user.name ${{ github.actor }}
```

Comment:
This should be using `Seneca-ICTOER` email and username too right?

Reply:
I think she was trying to deploy on vercel but she should make these changes

Reply:
> This should be using `Seneca-ICTOER` email and username too right?

thanks, I would like to keep it
`git config --global user.name ${{ github.actor }}` because it provides more flexibility than setting a fixed variable `Seneca-ICTOER`

Ref:
`github.actor` is the login of the user that initiated the workflow run (https://docs.github.com/en/actions/learn-github-actions/contexts)